Dockerfile

```
@@ -1,6 +1,7 @@
 FROM python:3.9
 
-RUN echo $(ip route | awk '{print $3}') > /docker_host_ip
+RUN set -o pipefail \
```

If you want to keep the pipefail, I found I had do to this:

```suggestion
RUN ["/bin/bash", "-c", "set -o pipefail && ip route | awk '{print $3}' > docker_host_ip"]
```

---

Right - if the current changes don't work I'll do this instead.